### DISCUSSION TITLE
Moving from discord to something else e.g. matrix

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [frictionlessdata/frictionlessdata.io#690] [Ideas] [answered]
Hi, there.

In 2020 people decided to move the Frictionless chat tool from Gitter to Discord. I never did go there, because I do not agree with Discord's terms of service and privacy policy (guess what, they are pretty bad, [Grade E](https://tosdr.org/en/service/536)). The discussion in which the matter was discussed was [in the old Frictionless forum](https://github.com/frictionlessdata/forum/issues/13), which is now closed. So I've decided to open a new one.

There are [alternatives with much better terms of service](https://herrmann.tech/en/blog/2021/08/10/my-discord-with-discord-or-choosing-alternatives-with-better-terms-of-service.html) and privacy policies which also have great features. [Matrix](https://matrix.org/) is an excellent candidate for that, as it is an open protocol with free and open source software implementations, like [Element](https://element.io/). Element has many of the same features as Discord, including the possibility to use voice and video chat. Matrix even features [bridges](https://matrix.org/bridges/), which allows the community administrators to mirror channels in other services, like Discord. People on Element can send and receive messages to people on Discord and vice versa, participate in all channels, etc. That way, the migration can go as slowly as people want – or even leave the bridge open if some people really prefer to keep using Discord.

@lwinfree, you [suggested](https://github.com/frictionlessdata/forum/issues/13#issuecomment-744657968) revisiting this idea on April 2021. We're now past that and it's August, so perhaps now is a good time?

**Comment by lwinfree** [comment]
Hi @augusto-herrmann thanks for bringing this up! We are actually planning on sending a survey next month that will include a question about Discord. We're hoping to get some feedback from that survey to help us understand what people want to communicate with. So this a very timely right now!

**Comment by loleg** [comment]
With a nod to Augusto's points, I want to mention that I find it easier to post "work" things on Slack or Mattermost, preferring Discourse (see also my [comment here](https://github.com/frictionlessdata/project/discussions/667#discussioncomment-1169086)) over both for more asynchronous and well-referenced interaction. 

Recently we set up a [Matterbridge](https://github.com/42wim/matterbridge) between the @opendatach Slack and Mattermost instances, which supports Matrix, Gitter and Discord as well. It seems to be working well, though there is a bit of inevitable confusion ("did person X on the other server really see my message?").

@lwinfree I am looking forward to the survey and evaluation!

**Comment by augusto-herrmann** [comment, marked as answer]
Now is as good time as any (especially now during the hackathon many more people are able to contribute), so I've gone ahead and created a Matrix Space for Frictionless (as I mentioned on #693):

https://matrix.to/#/#frictionless-data:matrix.org

Now all we need to do is to configure the **bridge** between the Matrix Space and the Discord server, and everything will be mirrored as if everyone was using the same service. Magic! :sparkles:  :purple_heart: :sparkles: 

In order to do that I've created an issue: #698

**Reply by augusto-herrmann** [reply]
Hi @lwinfree. How about we bridge a few more channels into the Matrix space this coming week?

**Reply by lwinfree** [reply]
HI @augusto-herrmann! Sorry I missed this earlier. Should we coordinate a time to do this together next week? I just tried to do it on my own, but it looks like the general channel isn't bridged correctly. There have been several messages in the general channel on discord, but they aren't viewable on matrix. Here's when I have time next week for a call: https://calendly.com/lilly-winfree/45min (I don't think it will take us 45 minutes!).

**Reply by augusto-herrmann** [reply]
I suspected as much, as it's been a long time without messages on Matrix. Thanks for being available to fix it and make this happen. For me it would be better any time on Thursday, but I'll send you an email for confirmation.

**Reply by augusto-herrmann** [reply]
I believe we can now mark this as solved, now that we have an effective bridge for Discord ↔ Matrix.

**Reply by lwinfree** [reply]
+1

**Reply by augusto-herrmann** [reply]
Now all we need is to create links pointing to the Matrix Space from the Frictionless Website. I've created an issue for it: [frictionless/website#264](https://github.com/frictionlessdata/website/issues/264).

**Comment by roll** [comment]
How does Slack look privacy/license/etc wise? Is it better than Discord?

**Reply by augusto-herrmann** [reply]
A lot better than Discord, for sure.

* Slack: privacy [grade B](https://tosdr.org/en/service/206)
* Discord: privacy [grade E](https://tosdr.org/en/service/536) (the worst possible grade)

I haven't compared them in detail, but I believe [Matrix.org's terms of use](https://tosdr.org/en/service/2455) are still more favorable to the user than Slack's.
